zuul-core/src/main/java/com/netflix/zuul/netty/server/BaseZuulChannelInitializer.java

```diff
@@ -71,7 +70,9 @@
 
 import java.util.List;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
```

Comment:
unused?

Reply:
Holdover from the uglier predecessor of this PR.  Removed!